plugins/jobs/kafka/jobs_kafka_test.go

```diff
@@ -797,6 +797,74 @@ func TestKafkaOTEL(t *testing.T) {
 	})
 }
 
+func TestKafkaPingFailed(t *testing.T) {
+	cont := endure.New(slog.LevelError)
+
+	cfg := &config.Plugin{
+		Version: "v2023.1.0",
+		Path:    "configs/.rr-kafka-ping-failed.yaml",
+		Prefix:  "rr",
+	}
+
+	l, _ := mocklogger.ZapTestLogger(zap.DebugLevel)
+	err := cont.RegisterAll(
+		cfg,
+		&server.Plugin{},
+		&rpcPlugin.Plugin{},
+		&jobs.Plugin{},
+		&kp.Plugin{},
+		&otel.Plugin{},
+		l,
+		&resetter.Plugin{},
+		&informer.Plugin{},
+	)
+	assert.NoError(t, err)
+
+	err = cont.Init()
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	_, err = cont.Serve()
+
+	assert.NotNil(t, err, "Failed: server should not run")
```

Comment:
```suggestion
	assert.Equals/Contains(t, err.Error(), "Failed: server should not run")
```

Reply:
`Serve` may not return an error

Reply:
Do you mean, that this assert might fail? 
```go
assert.NotNil(t, err, "Failed: server should not run")
```

Reply:
Yes, the test contains conditions under which the "Serve" should failed and give an error. If it doesn't, it's a bug.